packages/@webex/plugin-meetings/src/controls-options-manager/util.ts

```diff
@@ -12,9 +12,17 @@ const canUnsetMuteOnEntry = (displayHints: Array<string>): boolean =>
 const canUnsetDisallowUnmute = (displayHints: Array<string>): boolean =>
   displayHints.includes(DISPLAY_HINTS.DISABLE_HARD_MUTE);
 
+const canSetMuted = (displayHints: Array<string>): boolean =>
+  displayHints.includes(DISPLAY_HINTS.MUTE_ALL);
+
+const canUnsetMuted = (displayHints: Array<string>): boolean =>
+  displayHints.includes(DISPLAY_HINTS.UNMUTE_ALL);
+
 export default {
   canSetMuteOnEntry,
   canSetDisallowUnmute,
+  canSetMuted,
```

Comment:
can we change the name to canSetMuteAll or something 

someone could confuse this with local mute

Reply:
discussed offline